astropy/io/votable/tests/test_tree.py

```diff
@@ -175,3 +176,315 @@ def test_votable_tag():
     assert 'xmlns="http://www.ivoa.net/xml/VOTable/v1.3"' in xml
     assert 'xsi:schemaLocation="http://www.ivoa.net/xml/VOTable/v1.3 ' in xml
     assert 'http://www.ivoa.net/xml/VOTable/VOTable-1.4.xsd"' in xml
+
+
+def _squash_xml(data):
+    """
+    Utility squashing XML fragment to easier their comparison
+    This function is only used in the test module. It was more convenient for comparing the xml.
+    """
+    return data.replace(" ", "").replace("\n", "").replace('"', "").replace("'", "")
+
+
+def test_mivot_constructor():
+    """
+    Construct a MIVOT block with wrong tag to test the expected exception
+    """
+    with pytest.raises(ValueError, match="not well-formed"):
+        MivotBlock(
+            """
+            <VODML xmlns="http://www.ivoa.net/xml/mivot" >
+               <REPORT status="OK">Unit test mivot block1</REPORT>
+               <WRONG TAG>
+               </GLOBALS>
+            </VODML>
+            """
+        )
+
+
+def test_mivot_readout():
+    """
+    Test the MIVOT block extraction from a file against a reference block stored in data
+    """
+    votable = parse(get_pkg_data_filename("data/mivot_annnotated_table.xml"))
```

Comment:
The filename `mivot_annnotated_table.xml` has `nnn` instead of `nn` which is probably a typo.  If so, to avoid confusion, both the filename and this line should be fixed.